src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,45 @@
-export const PeopleTable = () => {
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+  findParent: (parentName: string) => string | JSX.Element;
+};
+
+export const PeopleTable: FC<Props> = ({
+  people,
+  findParent,
+}) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+  const { slug } = useParams();
+
+  const sortParams = (sortParam: string) => {
+    if (order && sort) {
+      return {
+        sort: null,
+        order: null,
+      };
+    }
+
+    if (sort) {
+      return {
+        sort: sortParam,
+        order: 'desc',
+      };
+    }
+
+    return {
```

Comment:
When the 'sort' parameter does not exist, you should assign 'sortParam' to 'sort' and set 'order' to 'asc' as default.